lib/videoClient.ts

```diff
@@ -192,64 +221,68 @@ const createMeeting = async (credential, calEvent: CalendarEvent): Promise<any>
   };
 
   const organizerMail = new VideoEventOrganizerMail(calEvent, uid, videoCallData);
-  const attendeeMail = new VideoEventAttendeeMail(calEvent, uid, videoCallData);
+  const attendeeMail = new VideoEventAttendeeMail(calEvent, uid, videoCallData, emailTemplates);
   try {
     await organizerMail.sendEmail();
   } catch (e) {
-    console.error("organizerMail.sendEmail failed", e)
+    console.error("organizerMail.sendEmail failed", e);
   }
 
   if (!creationResult || !creationResult.disableConfirmationEmail) {
     try {
       await attendeeMail.sendEmail();
     } catch (e) {
-      console.error("attendeeMail.sendEmail failed", e)
+      console.error("attendeeMail.sendEmail failed", e);
     }
   }
 
   return {
     uid,
-    createdEvent: creationResult
+    createdEvent: creationResult,
   };
 };
 
-const updateMeeting = async (credential, uidToUpdate: String, calEvent: CalendarEvent): Promise<any> => {
+const updateMeeting = async (credential, uidToUpdate: string, calEvent: CalendarEvent): Promise<any> => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689304)
```suggestion
const updateMeeting = async (credential, uidToUpdate: string, calEvent: CalendarEvent): Promise<unknown> => {
```